src/libraries/System.Runtime/tests/System.Globalization.Tests/DateTimeFormatInfo/DateTimeFormatInfoAbbreviatedDayNames.cs

```diff
@@ -28,6 +28,12 @@ public static IEnumerable<object[]> AbbreviatedDayNames_Set_TestData()
             yield return new object[] { new string[] { "", "", "", "", "", "", "" } };
         }
 
+        public static IEnumerable<object[]> AbbreviatedDayNames_Get_TestData_ICU()
+        {
+            yield return new object[] { new CultureInfo("en-US").DateTimeFormat, new string[] { "Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat" }, "en-US" };
```

Comment:
>new CultureInfo [](http://example.com/codeflow?start=40&length=15)

Use CultureInfo.GetCultureInfo instead. The reason is on Windows, avoid using user overrides.